desktop-widgets/mainwindow.cpp

```diff
@@ -1748,6 +1761,35 @@ void MainWindow::on_actionImportDiveLog_triggered()
 	}
 }
 
+void MainWindow::on_actionImportDiveSites_triggered()
```

Comment:
I will take another look at it, but I do want to use as much of the existing XML parsing as possible so that it remains consistent. The dive site export uses the same XML generation as the dive log export. I do think it is useful to be able to take a full dive log file and pull the sites out of it for import, but there may be a more efficient way to do it than your test function.

Reply:
> I do think it is useful to be able to take a full dive log file and pull the sites out of it for import

Fair enough. Speeding up the XML-parser is on my TODO list anyway. :)

Reply:
Actually, I think this is super-clever, even if this may not have been initially intended.
This way I can skip the whole "export all my sites" thing if I want to sync dive sites between two divers (let's say two buddies). I just import diver 1's XML divelog in dive site import mode and get all of their sites, but none of their dives.
Or am I missing something?

Reply:
@dirkhh I agree that this would allow another way to share all dive sites, but I think there will be times when a user may want to share all of their dive sites, but not all of the dive log and trip data. 

Ultimately what I would like to do is to remove the "dive sites" option from the export dialog altogether and add select boxes to the dive sites tab so that the user can select one, several, or all dive sites and hit an "Export selected dive sites" button. We could also move the "Import dive sites" trigger from the Import menu to a button on the dive sites tab to keep all of the dive site management functions in one place. But before we do that, I'd like to start getting some feedback from other users on the dive site import/export functions as they stand now.